layout/sections/Icons.tsx

```diff
@@ -0,0 +1,190 @@
+export const IconSeeMore = () => (
```

Comment:
We should refactor these icons in a common folder, which can be accessed globally.

Reply:
✅ Done